app/src/main/java/com/capstone/weatherapp/CityListFragment.kt

```diff
@@ -6,26 +6,33 @@ import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.capstone.weatherapp.databinding.FragmentCityListBinding
 
 class CityListFragment : Fragment() {
 
-    companion object {
-        fun newInstance() = CityListFragment()
-    }
-
     private lateinit var viewModel: CityListViewModel
```

Comment:
You can make it easier on yourself and avoid having to initialize the ViewModel on line 34 if you replaced this line with 
`private val viewModel: CityListViewModel by viewModels()`